src/otm_website_tests/core.clj

```diff
@@ -0,0 +1,12 @@
+(ns otm-website-tests.core)
+
+(def base-url (or (System/getenv "OTM_BASE_URL")
+                  "http://localhost:8080"))
+(def http-client-options
+  "Default options for clj-http."
+  {:follow-redirects false
+   :throw-exceptions false})
+
```

Comment:
Addressed in https://github.com/OpenTreeMap/otm-website-tests/commit/4baacf421f8d9b93c094d3f63d56696037483de7.